dali/python/nvidia/dali/data_node.py

```diff
@@ -91,6 +94,20 @@ def gpu(self) -> DataNode:
             return transferred_node
         return DataNode(self.name, "gpu", self.source)
 
+    def cpu(self) -> DataNode:
```

Comment:
The logic here is copied from .gpu(), see above.

Reply:
Yes, this would be great! Thanks

Reply:
I support above^^
And maybe `_to_backend` should not be internal, maybe people would like to use it directly (transfer by parameter and not the function name).